tutorials/nservicebus-101/nsb101-intro-paragraph.include.md

```diff
@@ -0,0 +1,3 @@
+The best way to get started with NServiceBus is to use it to build something realistic. In doing so you'll learn the architectural concepts behind the software, and start to learn its capabilities. In this step-by-step guide, you'll be building a back end for a retail e-commerce system. You'll learn how to send asynchronous messages between processes, how to use the Publish/Subscribe pattern to decouple business processes, and the advantages of using reliable messaging to enable automatic retries after processing failures.
```

Comment:
>In this step-by-step guide,

now its a guide and not a tutorial?

>building a back end for a retail e-commerce system

backend is one word.

>You'll learn how to send asynchronous messages between processes, how to use the Publish/Subscribe pattern to decouple business processes, and the advantages of using reliable messaging to enable automatic retries after processing failures.

For a new user who doesn't know NServiceBus or is new to async messages, this doesn't add value. Because the what is being described here. Nothing here indicates why all is this is important.